static/bidder-params/conversant.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Conversant Adapter Params",
+  "description": "A schema which validates params accepted by the Conversant adapter.",
+  "type": "object",
+  "properties": {
+    "site_id": {
+      "type": "string",
+      "description": "A Conversant specific ID which identifies the site."
+    },
+   	"secure": {
+   	  "type": ["boolean", "integer"],
```

Comment:
Now that `FlexBool` is gone, this is only `integer`.

Same goes for `mobile` below.

Reply:
Doh!  Thanks.